src/apache/conf/custom.conf

```diff
@@ -0,0 +1,42 @@
+#####################################################################################
+# Use this file to override variables set in the default Apache configuration
+# WARNING: This could break or slow down your Nextcloud instance or make it
+# impossible for some clients to connect
+#
+# If you need to include a file stored in your writable partition,
+# use ${SNAP_DATA} as the prefix
+#####################################################################################
+
+#########################
+#        GLOBAL         #
+#########################
+#
+# LogLevel: Control the number of messages logged to the error_log.
+# Possible values include: debug, info, notice, warn, error, crit,
+# alert, emerg.
+#
+# LogLevel warn
+
+#########################
+#          SSL          #
+#########################
+# Warning the Mac OSX client currently requires TLSv1
+#SSLProtocol All -SSLv3 -TLSv1 -TLSv1.1
+#SSLCipherSuite ECDHE-RSA-AES256-GCM-SHA384:ECDHE-RSA-AES128-GCM-SHA256:ECDHE-RSA-AES256-SHA384:ECDHE-RSA-AES128-SHA256
```

Comment:
Adding such things to this file is a bad idea. Remember you can't update it after it's installed, and I guarantee our recommended cipher suite will change over time. I suggest you simply put a link to the wiki with some sort of "see the following link for example configurations," and put this information on a wiki you can keep up-to-date.